server/apps/main/templates/main/home.html

```diff
@@ -20,135 +18,156 @@ <h1 class="big-heading">AutSPACEs</h1>
     <div class="row intro intro-text">
       <div class="col-lg-7">
         <h3 class="title-text">A space for autistic people to share our stories about our senses so we can build a better world for autistic people</h3>
-         <a class="col-lg-12" href="{% url 'main:about_us' %}">
-       <button class="btn btn-outline-light btn-lg float-left" type="button">Read More</button>
-   </a>
-
       </div>
+
       <div class="col-lg-5">
         <img class="title-image" src="{% static '/images/animation_a.jpg' %}" alt="Illustration" />
       </div>
     </div>
   </div>
-
 </section>
-<section id="value">
+
+<!-- Platform information -->
+<section id="platform_info">
   <div class="row">
-  <div class="col-lg-4 value-box">
-     <h3 class="value-heading">Increase</h3>
-  <p class="value-text">understanding of sensory processing</p>
+    <div class="col-lg-12 platform-title">
+      <h2 class="section-heading">About AutSPACEs</h2>
+    </div>
   </div>
+  <div class="row">
+    <div class= "col-lg-12 platform-info-text">
 
-  <div class="col-lg-4 value-box">
-  <h3 class="value-heading">Share</h3>
-  <p class="value-text">our strategies and experiences with each other</p>
-</div>
+      <p>AutSPACEs is made by and for autistic people, researchers, and allies
+      of autistic people, who think that the world would be a much better place
+      if autistic people had more of a say in how public and private spaces are
+      used and designed.</p>
 
-  <div class="col-lg-4 value-box">
-  <h3 class="value-heading">Advise</h3>
-  <p class="value-text">organisations and policy-makers for real-world impact</p>
-    </div>
+      <p>All autistic people are different. We want to collect together lots of
+      stories from autistic people, so that we can better understand how
+      sensory processsing differences affect the way autistic people navigate
+      the world around them.  Together, we can use this information to build a
+      better world for autistic people.</p>
 
-    <div class="col-lg-4 value-box">
-      <h3 class="value-heading">Educate</h3>
-      <p class="value-text">non-autistic people about autistic experiences</p>
-</div>
     </div>
+  </div>
+  <div class="row">
+    <div class="col float-right">
+      <a href="{% url 'main:about_us' %}">
+        <button class="btn btn-outline-primary btn-home-about btn-lg float-right" type="button">Read More</button>
+      </a>
+    </div>
+  </div>
 </section>
-<!--How to Use the Platform-->
+
+<!-- How to Use the Platform -->
 <section id="platform_usage">
   <div class="row">
-      <div class="col-lg-12 platform-title">
-      <h2 class="section-heading">How to Use The Platform</h2>
+    <div class="col-lg-12 platform-title">
+      <h2 class="section-heading">How to use the platform</h2>
```

Comment:
I don't remember on top-of-my-head what the user testing results where on this, but I do feel like we can potentially reorder and simplify this bit? I'd say the order 

1. View
2. (Learn)
3. Register
4. Share

could be an easier one, I don't think the edit necessarily needs to appear, and even the "learn" could go into "view"? 

Tagging @GeorgiaHCA in case she has a suggestion!